CriticalMass/ErrorHandler.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ErrorHandler.swift
+//  CriticalMaps
+//
+//  Created by Malte Bünz on 16.08.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import Foundation
+
+struct ErrorHandler {
+    static let `default` = ErrorHandler()
+
+    let genericErrorMessage = "Sorry, something went wrong"
+
+    func handleError(_ error: Error?) {
+        if let errorDescription = error?.localizedDescription {
+            presentToUser(message: errorDescription)
+        } else {
+            presentToUser(message: genericErrorMessage)
+        }
+    }
+
+    private func presentToUser(message: String) {
+        print(message)
```

Comment:
Do we want to add alerts in this case? Although we shouldn't present alerts if the app is running in the background

Reply:
I want to add something visual after this PR is merged. For now I am not sure how it looks like. Maybe also @zutrinken has an idea of error states